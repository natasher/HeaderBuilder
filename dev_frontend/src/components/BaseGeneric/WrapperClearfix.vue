<template>
  <div class="clearfix">
    <!-- @slot placement for clearfixed content -->
    <slot />
  </div>
</template>

<script>
export default {

  name: 'WrapperClearfix',

}
</script>
