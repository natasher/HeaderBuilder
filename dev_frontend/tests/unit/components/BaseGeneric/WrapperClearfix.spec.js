import { shallowMount } from '@vue/test-utils'
import WrapperClearfix from '../../../../src/components/BaseGeneric/WrapperClearfix.vue'

describe('WrapperClearfix.vue', () => {

  test('should match snapshot', () => {
    const wrapper = shallowMount(WrapperClearfix)

    expect( wrapper.element ).toMatchSnapshot()
  })

})