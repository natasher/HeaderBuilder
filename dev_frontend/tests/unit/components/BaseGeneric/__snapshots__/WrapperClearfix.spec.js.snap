// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WrapperClearfix.vue should match snapshot 1`] = `
<div
  class="clearfix"
/>
`;
